docs/troubleshooting.rst

```diff
@@ -42,6 +42,45 @@ For more information about the ``version`` tag please see: https://docs.datadogh
 
 To set ``service``, ``env``, and ``version`` properly for your environment, please see: https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes
 
+
+Root span is missing error details
+==================================
+
+If an error is raised during the execution of instrumented code, any span from instrumented code which does not handle the exception will include error details (error flag, message, type, traceback).
+
+However, if the instrumented code for a span handles the exception, that span will not include error details as the exception is not raised from the call stack of the span's execution context.
+
+This can be a problem for users who want to see error details from a child span in the root span of a trace. An example of this is when an error is raised during the execution of view code, subsequently handled by the web framework in order to return an error page, the error details from the span for an instrumented view will not be included in the root span for the request.
+
+While this is default behavior for integrations, users can add a trace filter to propagate the error details up to the root span::
+
+  from ddtrace import Span, tracer
+  from ddtrace.filters import TraceFilter
+
+
+  class ErrorFilter(TraceFilter):
+    def process_trace(self, trace):
```

Comment:
comment here describing the filter would be good.

e.g. find the first child span with an error and add it's error tags onto the root span.